wrappers/javascript/aries-askar-nodejs/babel.config.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path')
 
-const ariesAskarShared = require('../shared/package.json')
+const ariesAskarShared = require('../aries-askar-shared/package.json')
```

Comment:
I think it is needed for current Jest configuration. Maybe with some tweaks like in aries-framework-javascript setup it wouldn't be necessary anymore.

Reply:
I believe, but it might have been revised a couple of times, it was for importing from `aries-askar-shared` with hot reloading. Don't think it is still required, as we use import resolved. But this stuff always confuses me as we can do this in 100 different ways.

Reply:
OK, I've updated jest config to not need babel config file for NodeJS wrapper. I left it in react-native as I wasn't sure (had some errors when checking types) and will be updated in another PR anyway.